_sources/Debugging/toctree.rst

```diff
@@ -5,11 +5,7 @@ Debugging
    :maxdepth: 2
 
    WhatisDebugging.rst
-   intro-HowtobeaSuccessfulProgrammer.rst
```

Comment:
@presnick @laumurph I think there is some good stuff in either the How to be a successful programmer or the Beginning tips for debugging.

I think this extremely terse into misses an important point for students that 1. debugging is different than programming, and 2. that failure to write perfect code on the first try is very normal.  I think just jumping directly to the kinds of errors a bit harsh.  Can we restore one or more of those subsections?